lib/matplotlib/cm.py

```diff
@@ -38,14 +38,22 @@ def _gen_cmap_registry():
     for name, spec in datad.items():
         cmap_d[name] = (  # Precache the cmaps at a fixed lutsize..
             colors.LinearSegmentedColormap(name, spec, _LUTSIZE)
-            if 'red' in spec else
-            colors.ListedColormap(spec['listed'], name)
-            if 'listed' in spec else
-            colors.LinearSegmentedColormap.from_list(name, spec, _LUTSIZE))
+            if "red" in spec
+            else colors.ListedColormap(spec["listed"], name)
+            if "listed" in spec
+            else colors.LinearSegmentedColormap.from_list(name, spec, _LUTSIZE)
+        )
+
+        # Register colormap aliases for gray and grey.
+    cmap_d["grey"] = cmap_d["gray"]
+    cmap_d["Greys"] = cmap_d["Grays"]
```

Comment:
```suggestion
    cmap_d["Grays"] = cmap_d["Greys"]
```

This one, (confusingly, which is what this PR is addressing) is actually defined with an "e" already, so the "a" alias needs to be made, unlike the others.

This is causing import errors (and thus other CI errors)

Reply:
Oh okay, I will invert this aliasing and also fix the problems @oscargus mentioned above (it surely happened due to my editor and I just didn't noticed)